platforms/documentation/docs/src/docs/release/content/2025-10-05-tapi-help-version.adoc

```diff
@@ -0,0 +1,45 @@
+= Tooling API: Version info in IDEs
```

Comment:
What is this document? It doesn't seem to belong here at all.

Reply:
Exactly.